.github/workflows/v2-tests-e2e.yml

```diff
@@ -57,24 +65,34 @@ jobs:
         uses: bahmutov/npm-install@v1
         with:
           install-command: yarn
-      - name: Setup test-website project against master release
+
+      - name: Generate test-website project against master release
         run: |
-          KEEP_CONTAINER=true yarn test:build:v2
-          rm -rf node_modules
-      - name: Setup test-website project for Yarn v2
+          KEEP_CONTAINER=true yarn test:build:v2 -s
+          mv test-website ../test-website
+      - name: Install test-website project with Yarn Berry and nodeLinker = ${{ matrix.nodeLinker }}
         run: |
-          cd test-website
+          cd ../test-website
+
+          # we have to switch to berry first before setting the version we want
           yarn set version berry
+          # temporary using canary for #5342
+          yarn set version canary
```

Comment:
We've released 3.0.1 with the fix so this can be reverted

Reply:
ohhh too late :D 

anyway I don't mind staying on canary for now, I could report issues we have with it that do not happen in stable releases